settings.gradle

```diff
@@ -1,3 +1,11 @@
+// For the Nextflow plugin Gradle plugin
+pluginManagement {
+    repositories {
+        mavenLocal()
+        gradlePluginPortal()
+    }
+}
```

Comment:
This should not be needed anymore since the nextflow gradle plugin was published to Maven Central